exporter/elasticsearchexporter/exporter.go

```diff
@@ -238,6 +238,10 @@ func (e *elasticsearchExporter) pushMetricsData(
 						}
 					}
 				case pmetric.MetricTypeExponentialHistogram:
+					if metric.ExponentialHistogram().AggregationTemporality() == pmetric.AggregationTemporalityCumulative {
+						errs = append(errs, errors.New("dropping cumulative temporality exponential histogram"))
```

Comment:
```suggestion
						errs = append(errs, fmt.Errorf("dropping cumulative temporality exponential histogram %s", metric.Name()))
```
Maybe add metric name to the logs for a bit more data of what is being dropped so it could be acted on?